apis/nucleus/src/index.js

```diff
@@ -332,6 +332,44 @@ function nuked(configuration = {}) {
          * @typedef {function(function)} ReceiverFunction A callback function which receives another function as input.
          */
 
+        /**
+         * @typedef {object} SwitchButtonConfig
+         * @param {string} SwitchButtonConfig.option The option to be toggled. Currently supporting: 'checkboxes', 'search', 'toolbar', 'rangeSelect', 'dense'.
```

Comment:
I believe `SwitchButtonConfig.` should be omitted from the param names. For example, `SwitchButtonConfig.option` becomes `option`.

Running `yarn spec && yarn ts` in the `<repo>/apis/stardust` updates the spec and ts defs. It could be worth running them and have a look at them so they are correct.

Reply:
I think it could be done like this (at least that's how we do it in another repo) by adding a shorthand variable, e.g. `c`:
```
@typedef {object} c SwitchButtonConfig
@param {string} c.option
```
Good to know that command as well.